src/views/CosmicProxyView.tsx

```diff
@@ -2,21 +2,22 @@
 import * as React from 'react';
 import { useEffect } from 'react';
 import Select from 'react-select';
+import { VisynSimpleViewPluginType } from 'tdp_core';
 
-export function CosmicView({ parameters, onParametersChanged }: any) {
+type CosmicViewPluginType = VisynSimpleViewPluginType<{ currentId: string }>;
+
+export function CosmicView({ parameters, onParametersChanged }: CosmicViewPluginType['props']) {
   useEffect(() => {
     if (!parameters) {
       onParametersChanged({ currentId: '' });
     }
   });
 
-  console.log(parameters);
-
   return <iframe className="w-100 h-100" src="https://cancer.sanger.ac.uk/cosmic" />;
```

Comment:
Will you do the generalization and configuration of the proxy view in a follow-up PR?